boards/doiting/dt_bl10_devkit/dt_bl10_devkit.dts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2021-2025 ATL Electronics
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <bflb/bl60x.dtsi>
+#include "dt_bl10_devkit-pinctrl.dtsi"
+
+/ {
+	model = "2.4GHz Wi-Fi and BLE coexistence Module Development Kit";
+	compatible = "bflb,bl602";
+
+	chosen {
+		zephyr,flash = &flash0;
+		zephyr,itcm = &itcm;
+		zephyr,dtcm = &dtcm;
+		zephyr,sram = &sram0;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+	};
+};
+
+&cpu0 {
+	clock-frequency = <DT_FREQ_M(192)>;
+};
+
+&spi1 {
+	#address-cells = <1>;
+	#size-cells = <0>;
+	reg = <0x4000b000 0x1000 0x23000000 0xc00000>;
+
+	flash0: flash@0 {
+		compatible = "issi,is25lp128", "jedec,spi-nor";
+		status = "disabled";
```

Comment:
is this intentional?

Reply:
yes, this devices uses external flash memory and we need to define the node to make sure zephyr pick correct addresses. However, this can not be enabled because there is no driver yet available.